tailwind.config.js

```diff
@@ -0,0 +1,200 @@
+/** @type {import('tailwindcss').Config} */
+const { BREAKPOINT_REM } = require('./src/constants/page');
+const plugin = require('tailwindcss/plugin');
+
+export default {
+  content: ['./index.html', './src/**/*.{js,ts,jsx,tsx}'],
+  theme: {
+    screens: {
+      mobile: { max: BREAKPOINT_REM.mobile },
+      notMobile: { min: BREAKPOINT_REM.mobile },
+      tablet: { max: BREAKPOINT_REM.tablet },
+      notTablet: { min: BREAKPOINT_REM.tablet },
+      desktopSmall: { max: BREAKPOINT_REM.desktopSmall },
+      desktopMedium: { min: BREAKPOINT_REM.desktopMedium },
+      desktopLarge: { min: BREAKPOINT_REM.desktopLarge },
+    },
+    colors: {
```

Comment:
oh this is nifty

Reply:
yeah this is what allows shortcuts like `text-color-text-0` and `bg-color-layer-4` or just `bg-color-accent`